native/core/src/execution/datafusion/planner.rs

```diff
@@ -1692,16 +1692,33 @@ impl PhysicalPlanner {
             .and_then(|inner| inner.lower_frame_bound_struct.as_ref())
         {
             Some(l) => match l {
-                LowerFrameBoundStruct::UnboundedPreceding(_) => {
-                    WindowFrameBound::Preceding(ScalarValue::UInt64(None))
-                }
+                LowerFrameBoundStruct::UnboundedPreceding(_) => match units {
+                    WindowFrameUnits::Rows => {
+                        WindowFrameBound::Preceding(ScalarValue::UInt64(None))
+                    }
+                    WindowFrameUnits::Range | WindowFrameUnits::Groups => {
```

Comment:
Do we use `Groups`?

Reply:
We don't use `Groups`, but if we don't specify this case, `UpperFrameBoundStruct::UnboundedFollowing(_) => match units` fails with `pattern datafusion_expr::WindowFrameUnits::Groups not covered`.

Reply:
Can we add it as separate pattern branch and return error?